db/create_tables.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE poll(
+    id NUMERIC,
+    title VARCHAR(20) NOT NULL,
+    vote_until DATE DEFAULT CURRENT_DATE + INTERVAL '10 day',
+    CONSTRAINT poll_pk PRIMARY KEY (id)
+);
+
+CREATE TABLE poll_candidate(
+    poll_id NUMERIC,
+    candidate VARCHAR(30),
+    CONSTRAINT poll_candidate_pk PRIMARY KEY(poll_id, candidate),

```

Comment:
> 作成したテーブルのキーがあっているかどうか見てほしい(PRIMARY KEYとFOREIGN KEY）

まず前提として、正解があるわけではないです。
一つの投票に同じ候補名が重複しない前提なら `(poll_id, candidate)` の複合主キーでも良いと思います。

一方で、 Web アプリケーションで使用する DB ではサロゲートキーを設定することが多いです。
特に rails の active record migration では暗黙的に `id` カラムが主キーに設定されるようになっています。
この投票アプリを今後も継続的に開発すると考えたとき、候補名の重複を許す可能性がないと断言できないなら、サロゲートキーを設定するのが個人的には無難かなと思います。

```suggestion
    id bibserial,
    CONSTRAINT poll_candidate_pk PRIMARY KEY(id),
```